internal/commands/sbomtest/presenter.go

```diff
@@ -103,13 +111,33 @@ func resultToJSONOutput(res *snykclient.SBOMTestResult) JSONOutput {
 		}
 	}
 
-	sortIssues(vulns)
+	sortVulnerabilities(vulns)
+
+	lics := make([]LicenseIssue, 0, len(res.LicenseIssues))
+
+	for _, lic := range res.LicenseIssues {
+		for _, pkg := range lic.Packages {
+			lics = append(lics, LicenseIssue{
+				ID:          lic.ID,
+				PackageName: pkg.Name,
+				Name:        pkg.Name,
+				Version:     pkg.Version,
+
+				Title: lic.Title,
+
+				Severity: lic.SeverityLevel,
+			})
+		}
+	}
+
+	sortLicenseIssues(lics)
 
 	return JSONOutput{
 		OK:              res.Summary.TotalIssues == 0,
 		DependencyCount: len(res.Summary.Tested) + len(res.Summary.Untested),
-		Summary:         fmt.Sprintf("Found %d vulnerabilities", len(res.Vulnerabilities)),
+		Summary:         fmt.Sprintf("Found %d issues", len(res.Vulnerabilities)+len(res.LicenseIssues)),
```

Comment:
let's not do arithmetic in here, these numbers should come from the server. can we use `res.Summary.TotalIssues`?